ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactAccessibilityDelegate.java

```diff
@@ -96,6 +96,9 @@ private void scheduleAccessibilityEventSender(View host) {
   public enum AccessibilityRole {
     NONE,
     BUTTON,
+    CHECKEDTEXTVIEW,
```

Comment:
TalkBack does not make any announcement with this role.
Probably this is caused by the missing reactnative CheckedTextView Component ([example of checkedtextview](https://i.ytimg.com/vi/ZuGVJdicKdA/maxresdefault.jpg)).
I decided to remove the checkedtextview role.

Reply:
https://github.com/fabriziobertoglio1987/react-native/commit/45235d34e7519133f969147660b72fecd85e1c83